pkg/authorization/cache/policy_cache_test.go

```diff
@@ -0,0 +1,96 @@
+package cache
+
+import (
+	"testing"
+	"time"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	testregistry "github.com/openshift/origin/pkg/authorization/registry/test"
+)
+
+func TestPolicyGet(t *testing.T) {
+	policyRegistry := testregistry.NewPolicyRegistry(testPolicies(), nil)
+	bindingRegistry := testregistry.NewPolicyBindingRegistry(testBindings(), nil)
+
+	policyCache := NewPolicyCache(bindingRegistry, policyRegistry)
+	policyCache.Run(1 * time.Second)
+
+	time.Sleep(50 * time.Millisecond)
```

Comment:
For this sort of test, usually it's an integration.  Your unit test would not run go util.Forever loops, instead it would be incrementally triggering things and testing that they work.